src/main/java/org/simlar/simlarserver/services/createaccountservice/CreateAccountService.java

```diff
@@ -181,14 +181,16 @@ private AccountCreationRequestCount updateRequestTries(final SimlarId simlarId,
     }
 
     @SuppressWarnings("NonBooleanMethodNameMayNotStartWithQuestion")
-    private static void checkRequestTriesLimit(final int requestTries, final int limit, final String message) {
-        if (requestTries > limit) {
+    private static void checkRequestTriesLimit(final Integer requests, final int limit, final String message) {
+        final int requestTries = requests == null ? 0 : requests;
```

Comment:
`if (ObjectUtils.compare(requests, limits) >= 0) {`

Reply:
I choose ```ObjectUtils.defaultIfNull``` as we use ```requestTries``` twice.